Open Judge System/OJS.Services/Business/Contests/ContestsBusinessService.cs

```diff
@@ -158,7 +162,9 @@ public ServiceResult TransferParticipantsToPracticeById(int contestId)
         public void DeleteById(int id)
```

Comment:
Shouldn't the CRUD-related functionality (Delete in this case) reside in the Data rather than Business service?

Reply:
In this case the process of deletion requires some business logic also, except just deleting the entity.